cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -661,6 +663,40 @@ protected Collection<String> doGetScopes(BookId bookId) throws IOException {
 		return result;
 	}
 
+	@Override
+	protected CompletableFuture<CradleResultSet<String>> doGetScopesAsync(BookId bookId, Interval interval) throws CradleStorageException {
+		String queryInfo = String.format("Scopes in book %s from %s to %s",
+				bookId.getName(),
+				interval.getStart().toString(),
+				interval.getEnd().toString());
```

Comment:
`toString` will be called automatically. No need to explicitly call it
```suggestion
				interval.getStart(),
				interval.getEnd());
```